en/peripherals/t265_vio.md

```diff
@@ -0,0 +1,31 @@
+# T265 VIO Camera Setup Guide
+
+The [Intel Realsense Tracking Camera T265](https://www.intelrealsense.com/tracking-camera-t265/) is a [VIO](https://dev.px4.io/master/en/advanced/computer_vision.html) (or V-SLAM) camera produced by Intel.
+It provides odometry information which can be used to augment or replace other positioning systems on PX4.
+
+## Setup Instructions
+
+These instructions explain how to set up VIO using ROS (the supported method):
+
+- On the companion computer, install and configure [MAVROS](https://dev.px4.io/master/en/ros/mavros_installation.html) and verify it is connected successfully to the flight controller
+- Get the [VIO bridge ROS node from Auterion](https://github.com/Auterion/VIO_bridge)
+- Configure the VIO bridge launch file to reflect your camera orientation.
+  Default is with the lenses downwards facing.
+- Follow the instructions at [External position estimation](https://dev.px4.io/master/en/ros/external_position_estimation.html#ekf2-tuningconfiguration) for tuning the EKF2.
+
+## Troubleshooting
+
+First make sure MAVROS is able to connect successfully to the flight controller.
```

Comment:
How can you tell?

Reply:
The reason I put this here is because for avoidance a lot of people have problems because they didn't follow the MAVROS setup guide.